app/webpacker/components/Panel/PanelPages.jsx

```diff
@@ -169,7 +169,7 @@ export default {
   },
   [PANEL_PAGES.createNewComers]: {
     name: 'Create New Comers',
-    link: createNewComersUrl,
+    component: CreateNewComersPage,
```

Comment:
Not 100% sure but I think the correct capitalization is just "Newcomers".

Reply:
Oh yes, changed. (there are some places where it's still wrong, I'll change that in a separate PR after this is merged, because those are not related to migration)